polymer.externs.js

```diff
@@ -443,6 +443,13 @@ Polymer.Base.async = function(method, wait) {};
 */
 Polymer.Base.getPropertyInfo = function(property) {};
 
+/**
+ * @param {string} path Path that should be notified.
+ * @param {!Array<!PolymerSplice>} splices Array of splice records indicating
+ *     ordered changes that occurred to the array.                                                                                  
+ */
+Polymer.Base.notifySplices = function(path, splices) {};
```

Comment:
If I'm reading [the implementation](https://github.com/Polymer/polymer/blob/master/src/standard/notify-path.html#L425) correctly, this should really be on PolymerElement.prototype instead of Polymer.Base. I know it's confusing and not accurately representative of the real world, but in this file, Polymer.Base is really only for stuff that's being called statically that way.